Impossible-Agent-Suite/flash-loan/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "flash-loan-agent",
+  "version": "0.0.1",
+  "description": "Impossible Finance flash loan agent",
+  "scripts": {
+    "build": "tsc",
+    "start": "npm run start:dev",
+    "start:dev": "nodemon --watch src --watch forta.config.json -e js,ts,json  --exec \"npm run build && forta-agent run\"",
+    "start:prod": "forta-agent run --prod",
+    "tx": "npm run build && forta-agent run --tx",
+    "block": "npm run build && forta-agent run --block",
+    "range": "npm run build && forta-agent run --range",
+    "file": "npm run build && forta-agent run --file",
+    "publish": "forta-agent publish",
+    "push": "forta-agent push",
+    "disable": "forta-agent disable",
+    "enable": "forta-agent enable",
+    "keyfile": "forta-agent keyfile",
+    "test": "jest"
+  },
+  "dependencies": {
+    "forta-agent": "^0.0.35",
+    "forta-agent-tools": "^1.0.32"
```

Comment:
this should be a dev-dependency

Reply:
You're selected four lines here, but by looking at the outcome of a similar recommended change [here](https://github.com/NethermindEth/Forta-Agents/pull/111/files#diff-bfbd5e3d2ac2469ee61828991c4a035604b91dc7e3e4f7e05a24e637325ec9a3) I have only moved `forta-agent-tools` to `devDependencies`